prow/config.yaml

```diff
@@ -4149,3 +4149,10 @@ periodics:
       secret:
         defaultMode: 256
         secretName: ssh-key-secret
+trigger:
+- repos:
+  - kubernetes
+  - kubernetes-incubator
```

Comment:
Both work. Same as in `plugins.yaml`, you can specify an org or a specific repo.